package.json

```diff
@@ -15,14 +15,13 @@
     "prepack": "mv src/package.json src_package.json",
     "postpack": "mv src_package.json src/package.json",
     "start": "webpack-dev-server --config configs/webpack.dev.config.js",
-    "start:https": "webpack-dev-server --config configs/webpack.dev.config.js --https --pfx configs/webpack.docs.dev.pfx",
-    "serve:e2e": "lite-server -c=e2e/bs-config.e2e.json",
-    "e2e": "concurrently \"npm run serve:e2e\" \"grunt protractor:e2e\" --kill-others --success first",
-    "build:library": "ng-packagr -p ./src/ng-package.json",
-    "build:documentation:prod": "node --max-old-space-size=4096 ./node_modules/webpack/bin/webpack.js --colors --config ./configs/webpack.docs.prod.config.js",
-    "package:library": "grunt build:library",
-    "lint": "grunt lint",
-    "e2e:develop": "concurrently \"webpack --watch --progress --colors --config configs/webpack.e2e.config.js\" \"tsc -w --project .\\e2e\\tsconfig.json\" \"npm run serve:e2e\""
+    "start-https": "npm run start -- --https --pfx configs/webpack.docs.dev.pfx",
+    "e2e": "concurrently \"npm run e2e-serve\" \"grunt protractor:e2e\" --kill-others --success first",
+    "e2e-serve": "lite-server -c=e2e/bs-config.e2e.json",
+    "e2e-develop": "concurrently \"webpack --watch --progress --colors --config configs/webpack.e2e.config.js\" \"tsc -w --project .\\e2e\\tsconfig.json\" \"npm run e2e-serve\"",
+    "library-build": "ng-packagr -p ./src/ng-package.json",
+    "grunt-build": "grunt build",
+    "grunt-build-library": "grunt build:library"
```

Comment:
Without the former build:library task, these can just be "build" and "build:library".